components/inputs/input-date-time.js

```diff
@@ -0,0 +1,134 @@
+import './input-date.js';
+import './input-fieldset.js';
+import './input-time.js';
+import { convertLocalToUTCDateTime, convertUTCToLocalDateTime } from '@brightspace-ui/intl/lib/dateTime.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { formatDateInISO, formatTimeInISO, parseISODate, parseISODateTime, parseISOTime } from '../../helpers/dateTime.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+export function formatDateTimeInISO(val) {
+	if (!val) {
+		throw new Error('Invalid input: Expected input to be an object');
+	}
+	return `${formatDateInISO({year: val.year, month: val.month, date: val.date})}T${formatTimeInISO({hours: val.hours, minutes: val.minutes, seconds: val.seconds})}.000Z`;
+}
+
+class InputDateTime extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			disabled: { type: Boolean },
+			label: { type: String },
```

Comment:
Since there's no way to properly label things from outside the component, I think we should `console.warn` if a label isn't provided -- just like time input does. Same in `date-input`.